style/ruby/.rubocop.yml

```diff
@@ -467,6 +467,9 @@ Style/InitialIndentation:
   Description: >-
     Checks the indentation of the first non-blank non-comment line in a file.
   Enabled: false
+  
+Style/SymbolArray:
```

Comment:
Sometimes from the documentation, but what you wrote in the commit is fine: `changes [ :abc, :123] to %i[abc 123]`